### PR TITLE
Stand up basic website

### LINKED ISSUE #3: Stand up basic website
As a user, I want [SpokaneTech.org](https://SpokaneTech.org) to resolve to a website. Currently it resolves to nothing :)
- [x] Harden security -> https://docs.djangoproject.com/en/5.0/howto/deployment/checklist/
- [x] Need a webapp to deploy
- [x] Configure domain registrar (@joeriddles likely needs to do this)

### PR DESCRIPTION
**Pull request by KGB33**
Closes #3 

- [ ] Harden security -> https://docs.djangoproject.com/en/5.0/howto/deployment/checklist/
  - [x] Run (and fix) `manage.py check --deploy`
  - [ ] HTTP**s** configuration
- [ ] Need a webapp to deploy
  - [ ] Database Configuration
- [ ] GitHub Actions
  - [ ] (If Containerized) Automated Build.
  - [ ] Auto-deploy to chosen cloud provider.
- [ ] Configure domain registrar (Can be done after merged.)

### CODE REVIEW

File: src/spokanetech/settings.py [hunk 1]
```diff
@@ -19,65 +20,86 @@
...
+        ) from e
+
+    DEBUG = False
+    ALLOWED_HOSTS = ["spokanetech.org"]
+
+    # SSL Options
+    # TODO: These will have to change depending on how infra-platform handles SSL termination
+    # https://docs.djangoproject.com/en/5.0/ref/settings/#secure-hsts-preload
+    SECURE_HSTS_SECONDS = 0
+    SECURE_SSL_REDIRECT = False
```
Comment (joeriddles) [inline src/spokanetech/settings.py:47]: We might want to wait to turn this on until the site is stood up:

> Setting this incorrectly can irreversibly (for some time) break your site. Read the [HTTP Strict Transport Security](https://docs.djangoproject.com/en/5.0/ref/middleware/#http-strict-transport-security) documentation first.
> — https://docs.djangoproject.com/en/5.0/ref/settings/#secure-hsts-seconds

**Review by joeriddles — APPROVED** [review]

File: src/spokanetech/settings.py (hunk 1 above)
Comment (KGB33) [reply src/spokanetech/settings.py:47]: Agreed. I set these to the default/off values for now.

Comment (joeriddles) [reply src/spokanetech/settings.py:47]: Oh, duh!

**Comment by joeriddles** [comment]
Looks good, feel free to merge.

**Pull request merged**